app/src/test/java/org/simple/clinic/user/UserSessionTest.kt

```diff
@@ -928,4 +954,127 @@ class UserSessionTest {
         )
     )
   }
+
+  @Test
+  fun `when the login happens, set the user information in analytics`() {
```

Comment:
Large test!

Reply:
Yes, it's a consequence of how the login process is implemented. I'll try seeing if I can fix this. Otherwise, I'll improve it when breaking up `UserSession`.

Reply:
I'll do this when refactoring `UserSession`.